crates/mun_codegen/src/ir/body.rs

```diff
@@ -69,6 +70,7 @@ impl<'a, 'b, D: IrDatabase> BodyIrGenerator<'a, 'b, D> {
             function_map,
             dispatch_table,
             active_loop: None,
+            hir_function,
```

Comment:
What function is this? The parent function that this body is part of?

Reply:
This is the HIR representation of the function that the body is part of. Its required to determine the return type.

Reply:
Okay. I was wondering whether there was a more verbose way of naming `hir_function`, but cannot quickly come up with anything.